go/engine/context.go

```diff
@@ -21,6 +23,7 @@ type Context struct {
 	UpdateUI    libkb.UpdateUI
 
 	LoginContext libkb.LoginContext
+	NContext     context.Context
```

Comment:
I think the name `NContext` is very cryptic, and it's something that you'll be spreading all throughout the code. IT's not a local variable.